### PR TITLE
Add get-an-issue-comment API endpoint

### PR DESCRIPTION
**Pull request by onukura**
### Before submitting a pull-request to GitBucket I have first:

- [x] read the [contribution guidelines](https://github.com/gitbucket/gitbucket/blob/master/.github/CONTRIBUTING.md)
- [x] rebased my branch over master
- [x] verified that project is compiling
- [ ] verified that tests are passing
- [x] squashed my commits as appropriate *(keep several commits if it is relevant to understand the PR)*
- [ ] [marked as closed using commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) all issue ID that this PR should correct

## About this PR

This PR add "get-an-issue-comment" [API endpoint](https://developer.github.com/v3/issues/comments/#get-an-issue-comment).

### CODE REVIEW

File: src/main/scala/gitbucket/core/controller/api/ApiIssueCommentControllerBase.scala [hunk 1]
```diff
@@ -35,9 +35,20 @@ trait ApiIssueCommentControllerBase extends ControllerBase {
    */
 
   /*
-   * iii. Get a single comment
-   * https://developer.github.com/v3/issues/comments/#get-a-single-comment
+   * iii. Get an issue comment
+   * https://developer.github.com/v3/issues/comments/#get-an-issue-comment
```
Comment (SIkebe) [inline src/main/scala/gitbucket/core/controller/api/ApiIssueCommentControllerBase.scala:39]:
```suggestion
   * https://docs.github.com/en/rest/reference/issues#get-an-issue-comment
```

**Review by SIkebe — COMMENTED** [review]
Get-an-issue-comment API in GitHub returns single issue-comment like this.
```
{
  "id": 1,
  "node_id": "MDEyOklzc3VlQ29tbWVudDE=",
  "url": "https://api.github.com/repos/octocat/Hello-World/issues/comments/1",
  "html_url": "https://github.com/octocat/Hello-World/issues/1347#issuecomment-1",
  "body": "Me too",
  "user": {
    "login": "octocat",
    "id": 1,
    "node_id": "MDQ6VXNlcjE=",
    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
    "gravatar_id": "",
    "url": "https://api.github.com/users/octocat",
    "html_url": "https://github.com/octocat",
    "followers_url": "https://api.github.com/users/octocat/followers",
    "following_url": "https://api.github.com/users/octocat/following{/other_user}",
    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/octocat/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
    "organizations_url": "https://api.github.com/users/octocat/orgs",
    "repos_url": "https://api.github.com/users/octocat/repos",
    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
    "received_events_url": "https://api.github.com/users/octocat/received_events",
    "type": "User",
    "site_admin": false
  },
  "created_at": "2011-04-14T16:00:49Z",
  "updated_at": "2011-04-14T16:00:49Z"
}
```

However the one in GitBucket added in this PR returns an array of comments.
(Of course, this array always include only one entity)
```
[
    {
        "id": 18,
        "user": {
            "login": "root",
            "email": "root@example.com",
            "type": "User",
            "site_admin": true,
            "created_at": "2018-06-03T06:59:36Z",
            "id": 0,
            "url": "http://localhost:8080/api/v3/users/root",
            "html_url": "http://localhost:8080/root",
            "avatar_url": "http://localhost:8080/root/_avatar"
        },
        "body": "This is a comment by root.",
        "created_at": "2018-10-24T08:16:29Z",
        "updated_at": "2018-10-24T08:16:29Z",
        "html_url": "http://localhost:8080/root/test1/issues/1#comment-18"
    }
]
```

This difference may cause compatibility issues.

**Comment by SIkebe** [comment]
1. If the specified comment-id is valid, GitBucket returns the comment info.
image

1. If the specified username or repository does not exist, GitBucket returns `Not Found`.
image

-----

Above two cases are OK, but if the specified comment-id does not exist in the repository or associates with the action other than `comment`, GitBucket returns nothing.
image

Is this behavior the same as GItHub?

**Comment by onukura** [comment]
@SIkebe Thanks for your review!
> Above two cases are OK, but if the specified comment-id does not exist in the repository or associates with the action other than comment, GitBucket returns nothing.

that was because I forgot to handle case that there is not comment for specified `id`... 😅  I fixed this problem and response format from array to json in https://github.com/gitbucket/gitbucket/pull/2536/commits/a37967d6e305c463633d56fe24e0d1ba432a4e61

**Comment by takezoe** [comment]
@onukura Thanks for working on this! Could you also add this API to https://github.com/gitbucket/gitbucket/wiki/API-WebHook with `(since 4.35.0)` notation?

**Comment by onukura** [comment]
@takezoe updated wiki. Thanks for merging this PR!

**Pull request merged**